Library/Formula/pdf2htmlex.rb

```diff
@@ -20,6 +19,11 @@ class Pdf2htmlex < Formula
     url "https://github.com/coolwanglu/fontforge.git", :branch => "pdf2htmlEX"
   end
 
+  resource "AdobeXMLFormsSamples.pdf" do
+    url "http://partners.adobe.com/public/developer/en/xml/AdobeXMLFormsSamples.pdf"
+    sha256 "36547f22533af1d676e8beb4de56a206fff397481bf95fc9c95924cd8184ff8a"
+  end
```

Comment:
Can it be replace with the pdf file stored in test features? Or it has some unique requirement.